modules/custom/az_paragraphs/az_paragraphs_photo_gallery/templates/paragraph--az-photo-gallery.html.twig

```diff
@@ -105,10 +105,17 @@
                       {{ imagestyle }}
                     </div>
 
-                    {# Render the caption, if one exists. #}
-                    {% if item.entity.field_az_caption.value %}
+                    {# Render the caption/credit area, if one exists. #}
+                    {% if (item.entity.field_az_caption.value) or (item.entity.field_az_credit.value) %}
                       <div class="carousel-caption{% if grid %} az-gallery-caption d-block{% else %} d-none d-md-block{% endif %}">
-                        <p>{{ item.entity.field_az_caption.value }}</p>
+                        {# Render the caption, if one exists. #}
+                        {% if item.entity.field_az_caption.value %}
+                          <p>{{ item.entity.field_az_caption.value }}</p>
```

Comment:
Can we consider removing the bottom margin here if the caption is alone? Having a caption only adds extra bottom spacing.

Reply:
Agreed, sounds good to me.